src/main.rs

```diff
@@ -359,6 +359,9 @@ fn start_vmm(cmd_arguments: ArgMatches) {
 }
 
 fn main() {
+    // Ensure all created files (.e.g sockets) are only accessible by this user
+    let _ = unsafe { libc::umask(0o077) };
```

Comment:
@rbradford shouldn't this be `700` instead of `077`?

Reply:
Its a mask: so input &~umask e.g. 0644 &~ 0077 = 0600

Reply:
Definitely missed that :)